src/main/java/ch/njol/skript/effects/EffTitle.java

```diff
@@ -0,0 +1,96 @@
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.effects;
+
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.util.Timespan;
+import ch.njol.util.Kleenean;
+
+@Name("Title")
+@Description("Sends a title/subtitle to the given player(s) with optional fadein/stay/fadeout times")
+@Examples({"send title \"Competition Started\" with subtitle \"Have fun, Stay safe!\" to player for 5 seconds",
+		"send title \"Hi %player%\" to player", "send title \"Loot Drop\" with subtitle \"starts in 3 minutes\" to all players",
+		"send title \"Hello %player%!\" with subtitle \"Welcome to our server\" to player with fadein 1 second for 5 seconds with fadeout 1 second"})
+@Since("INSERT VERSION")
+public class EffTitle extends Effect {
+	
+	static {
+		Skript.registerEffect(EffTitle.class, "send [the] title %string% [with subtitle %-string%] to %players% [with fade[(-| )]in %-timespan%] [for %-timespan%] [with fade[(-| )]out %-timespan%]");
+	}
+	
+	@SuppressWarnings("null")
+	private Expression<String> title;
+	@Nullable
+	private Expression<String> subtitle;
+	
+	@SuppressWarnings("null")
+	private Expression<Player> recipients;
+	@SuppressWarnings("null")
+	private Expression<Timespan> fadein;
+	@SuppressWarnings("null")
+	private Expression<Timespan> stay;
+	@SuppressWarnings("null")
+	private Expression<Timespan> fadeout;
+	
+	@SuppressWarnings({"unchecked", "null"})
+	@Override
+	public boolean init(final Expression<?>[] exprs, final int matchedPattern, final Kleenean isDelayed, final SkriptParser.ParseResult parser) {
+		title = (Expression<String>) exprs[0];
+		subtitle = (Expression<String>) exprs[1];
+		recipients = (Expression<Player>) exprs[2];
+		fadein = (Expression<Timespan>) exprs[3];
+		stay = (Expression<Timespan>) exprs[4];
+		fadeout = (Expression<Timespan>) exprs[5];
+		
+		return true;
+	}
+	
+	@SuppressWarnings("null")
+	@Override
+	protected void execute(final Event e) {
+		String msg1 = title.getSingle(e);
+		String msg2 = subtitle != null ? subtitle.getSingle(e) : null;
+		Integer int1 = fadein != null ? (int) fadein.getSingle(e).getTicks_i() : 10;
```

Comment:
why call it int1? you also don't need the wrapper type or the int cast

Reply:
I guess the int1 servers no purpose other than just having the three in a row all named to match
as for the (int) part, when I remove it, i get a warning telling me it requires an int but found a long

Reply:
but why not just use names that actually make sense

Reply:
Okay the names have been updated, from int1,2,3 to fin, sty, fout
Hopefully that is more to your liking :)
If not, any suggestions are welcome, clearly I would love to use fadein, stay, fadeout, but those are already taken!

Reply:
why not the whole name fadein fadeout etc

Reply:
Because unless I'm completely stupid, I cant do this?!?!
```
Interger fadein = fadein
```

Reply:
int fadein = this.fadein == null ? 70 : this.fadein(...)